Sources/SwiftDocC/Model/Rendering/Variants/JSONPatchApplier.swift

```diff
@@ -25,20 +25,18 @@ public struct JSONPatchApplier {
     /// - Returns: The JSON data with the patch applied.
     /// - Throws: This function throws an ``Error`` if the application was not successful.
     public func apply(_ patch: JSONPatch, to jsonData: Data) throws -> Data {
-        let json = try JSONDecoder().decode(JSON.self, from: jsonData)
+        var json = try JSONDecoder().decode(JSON.self, from: jsonData)
```

Comment:
Are you slipping in a performance fix here? Was there a bug earlier? Or is this just a coding style change?

Reply:
Yes, this was a small performance fix for something that I stumbled upon. If you prefer I can extract it into a separate PR. I just find that I tend to forget if I don't fix things that I stumble upon.